airgun/views/host_new.py

```diff
@@ -294,7 +368,19 @@ class InstallPackagesView(View):
     cancel = Button('Cancel')
 
 
+class EditAnsibleRolesView(View):
+    """Edit Ansible Roles Modal"""
+
+    ROOT = ''
+    #No current representation for this Widget in Widgetastic
```

Comment:
Are you planning to write this widget in upstream Widgetastic?

Reply:
I feel like I will have to, if this is going to be automated in the same was as everything else. Perhaps there's an option to reach out to dev and get some assistance.